e2e/tests/adminManagesOrders_test.js

```diff
@@ -258,14 +268,18 @@ Scenario('Interim supervision order (C35B)', async ({I, caseViewPage, manageOrde
   });
 });
 
-Scenario('Create C47A appointment of a Children\'s Guardian', async ({I, caseViewPage, manageOrdersEventPage}) => {
+Scenario('Create C47A appointment of a Children\'s Guardian', async ({I, caseViewPage, manageOrdersEventPage, uploadAdditionalApplicationsEventPage}) => {
+  //Add additional applications
+  await manageDocumentsForLAHelper.uploadC2(I, caseViewPage, uploadAdditionalApplicationsEventPage);
```

Comment:
can maybe use the prepared data rather than using the ui to add the c2? (speeding up)

Reply:
I will create a tech debt ticket for this one.